Makefile

```diff
@@ -0,0 +1,5 @@
+test-metrics-php-laravel:
```

Comment:
Added these as convenience methods because running this through the `docker-`compose flow during a painful debugging session it was extremely frustrating having rebuild the Docker whenever I had problems.